cases-SEP31/sep10.test.js

```diff
@@ -192,7 +192,7 @@ describe("SEP10", () => {
         expect(tokenJson.error, logs).toBeTruthy();
       });
 
-      it("fails if the signed challenge isn't signed by the servers SIGNING_KEY", async () => {
+      it.skip("fails if the signed challenge isn't signed by the servers SIGNING_KEY", async () => {
```

Comment:
Not needed right now, but we need to build our own challenge transaction instead of trying to clear the signatures on the anchor's challenge. It looks like the JS SDK blocks you from writing to the signatures attribute. Our challenge transaction also needs to pass the checks in [readChallengeTx](https://stellar.github.io/js-stellar-sdk/libdocs_utils.js.html)